src/Template/Saefis/xml/e2b.ctp

```diff
@@ -321,10 +325,10 @@
             </drug>
             <?php  endforeach; ?>
             <summary>
-                <narrativeincludeclinical><?php echo $saefi['signs_symptoms']; ?></narrativeincludeclinical>
-                <reportercomment><?php echo $saefi['relevant_findings']; ?></reportercomment>
+                <narrativeincludeclinical><?php echo preg_replace('/[^A-Za-z0-9\s+]/', '',$saefi['signs_symptoms']); ?></narrativeincludeclinical>
+                <reportercomment><?php echo preg_replace('/[^A-Za-z0-9\s+]/', '',$saefi['relevant_findings']); ?></reportercomment>
                 <senderdiagnosismeddraversion>WHO-ART</senderdiagnosismeddraversion>
-                <senderdiagnosis><?php echo $saefi['past_drug_therapy']; ?></senderdiagnosis>
+                <senderdiagnosis><?php echo preg_replace('/[^A-Za-z0-9\s+]/', '',$saefi['past_drug_therapy']); ?></senderdiagnosis>
```

Comment:
Create a utility method that will perform the replace function and reuse it as appropriate